src/layouts/BasicLayout.js

```diff
@@ -280,19 +280,22 @@ class BasicLayout extends React.PureComponent {
                 {
                   key: 'Pbench Dashboard',
                   title: 'Pbench Dashboard',
-                  href: 'https://example.com',
+                  href:
```

Comment:
`datastoreConfig.results` would give us an endpoint of `/results/dashboard` here which would route the user to an unknown page. We can consider removing this key altogether as the user is already at the Pbench Dashboard link.

Reply:
fixed